src/utils/index.ts

```diff
@@ -4,3 +4,4 @@ export {KeyMap, KeyCode} from './key-map';
 export {default as getLogger} from './logger';
 export {withKeyboardA11y} from './popup-keyboard-accessibility';
```

Comment:
is converting the other places in the player that use humanizeDuration to use this method instead part of this ticket ? If not, we should have a separate ticket for this

Reply:
this is not part of the ticket. i can create a FEC ticket for that.